Description: No X11R6 Project Root
 We must not overlap with any existing files, therefore, both
 /usr/X11R6 and /usr are excluded possibilities for ProjectRoot.
 Use %_libexecdir/somename, since that is still allowed by FHS.
Forward: pending
Author: Jan Engelhardt <jengelh@medozas.de>
Last-Update: 2012-01-27 01:35:49.164415307 +0100
---
 nx-X11/config/cf/site.def |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: nx-X11/config/cf/site.def
===================================================================
--- a/nx-X11/config/cf/site.def
+++ b/nx-X11/config/cf/site.def
@@ -72,7 +72,7 @@ XCOMM site:  $XFree86: xc/config/cf/site
 #ifdef AfterVendorCF
 
 #ifndef ProjectRoot
-#define ProjectRoot /usr/X11R6
+#define ProjectRoot /usr/local/lib/nx
 #endif
 
 /*
